app/views/spina/registers/show.html.haml

```diff
@@ -33,12 +33,14 @@
 
       .govuk-metadata
         %dl
-          %dt Updated:
-          %dd
-            = DateTime.parse(@register_data.get_entries.last[:timestamp]).strftime('%d/%m/%Y')
           %dt Managed by:
           %dd
             = link_to @register.authority, "https://www.gov.uk/government/organisations/#{@register.authority.parameterize}", target: :blank
+          %dt Updated:
+          %dd
+            = DateTime.parse(@register_data.get_entries.last[:timestamp]).strftime('%d/%m/%Y')
```

Comment:
Could we move this up to the controller, as a function `formatted_timestamp` or similar?